ria_remote/create_sibling_ria.py

```diff
@@ -0,0 +1,305 @@
+        if ssh_host == '0':
+            ssh_host = None
+        lgr.info("create siblings '{}' and '{}' ...".format(sibling, storage_sibling))
+
+        lgr.debug('init special remote {}'.format(storage_sibling))
+        cmd = ['git', 'annex',
+               'initremote', storage_sibling,
+               'type=external',
+               'externaltype=ria',
+               'encryption=none',
+               'autoenable=true'
+               ]
+        result = subprocess.run(cmd, cwd=str(ds.path), stderr=subprocess.PIPE)
+        if result.returncode != 0:
+            if force and result.stderr.startswith(b'git-annex: There is already a special remote named'):
+                # run enableremote instead
+                cmd[2] = 'enableremote'
+                subprocess.run(cmd, cwd=str(ds.path))
+            else:
+                yield get_status_dict(
+                    status='error',
+                    message="initremote failed.\nstdout: %s\nstderr: %s" % (result.stdout, result.stderr),
+                    **res_kwargs
+                )
+                return
+
+        # 1. create remote object store:
+        # Note: All it actually takes is to trigger the special remote's `prepare` method once.
+        # ATM trying to achieve that by invoking a minimal fsck.
+        # TODO: - It's probably faster to actually talk to the special remote (i.e. pretending to be annex and use the
+        #         protocol to send PREPARE)
+        #       - Alternatively we can create the remote directory and ria version file directly, but this means code
+        #         duplication that then needs to be kept in sync with ria-remote implementation.
+        #       - this leads to the third option: Have that creation routine importable and callable from
+        #         ria-remote package without the need to actually instantiate a RIARemote object
+        lgr.debug("initializing object store")
+        subprocess.run(['git', 'annex', 'fsck', '--from={}'.format(storage_sibling), '--fast', '--exclude=*/*'],
+                       cwd=str(ds.path))
+
+        # 2. create a bare repository in-store:
+        # determine layout locations
+        repo_path, archive_path, objects_path = RIARemote.get_layout_locations(base_path, ds.id)
+
+        lgr.debug("init bare repository")
+        # TODO: we should prob. check whether it's there already. How?
+        # Note: like the special remote itself, we assume local FS if no SSH host is specified
+        disabled_hook = sh_quote(str(repo_path / 'hooks' / 'post-update.sample'))
+        enabled_hook = sh_quote(str(repo_path / 'hooks' / 'post-update'))
+
+        if ssh_host:
+            from datalad import ssh_manager
+            ssh = ssh_manager.get_connection(ssh_host, use_remote_annex_bundle=False)
+            ssh.open()
+            ssh('cd {} && git init --bare'.format(sh_quote(str(repo_path))))
```

Comment:
Do we need to be able to send additional options? `--shared`?

Reply:
Support `create-sibling --shared {false|true|umask|group|all|world|everybody|0xxx}`

Reply:
there is also `--group GROUP` in `create-sibling`

Reply:
Done